docs/UserGuide.md

```diff
@@ -542,25 +651,27 @@ HRMate data are saved automatically as a JSON file `[JAR file location]/data/hrm
+| **Edit**            | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                           |
+| **View tag**        | `view-tag`                                                                                                                                                            |
+| **Add tag**         | `add-tag EMPLOYEE_LIST_INDEX TAG`<br> e.g., `add-tag 3 remote`                                                                                                        |
+| **Delete tag**      | `delete-tag EMPLOYEE_LIST_INDEX TAG`<br> e.g., `delete-tag 3 remote`                                                                                                  |
+| **Find all tags**   | `find-all-tag [t/TAG]...`<br> e.g.,`find-all-tag t/remote t/full-time`                                                                                                |
+| **Find some tags**  | `find-some-tag [t/TAG]...`<br> e.g.,`find-some-tag t/remote t/full-time`                                                                                              |
+| **Find**            | `find SEARCH_QUERY...​`<br> e.g., `find James Jake`                                                                                                                   |
+| **Help**            | `help`                                                                                                                                                                |
+| **List**            | `list`                                                                                                                                                                |
+| **Import Contacts** | `import`                                                                                                                                                              |
+| **Export Contacts** | `export FILE_NAME`                                                                                                                                                    |           
+| **Import Leaves**   | `import-leaves`                                                                                                                                                       |
+| **Export Leaves**   | `export-leaves FILE_NAME`                                                                                                                                             |           
+| **Edit leave**      | `edit-leave INDEX [title/TITLE] [start/START_DATE] [end/END_DATE] [d/DESCRIPTION] [s/STATUS]`<br> e.g., `edit-leave 1 title/medical leave start/2023-11-01`           |
+ | **Delete leave**    | `delete-leave LEAVE_LIST_INDEX`<br> e.g., `delete-leave 1`                                                                                                            |
```

Comment:
i noticed that find-leave-status and find-leave-range are not inside the command summary.

Reply:
Good catch, I've resolved it, thanks!